astropy/modeling/fitting.py

```diff
@@ -1148,7 +1137,6 @@ def __init__(self, calc_uncertainties=False, use_min_max_bounds=True):
         self.fit_info = None
         self._calc_uncertainties = calc_uncertainties
         self._use_min_max_bounds = use_min_max_bounds
-        super().__init__()
```

Comment:
Is this related?

Reply:
yes. This line does nothing on `main` (because there's no actual parent class), but it creates failures now that `Fitter` is a parent class, since `Fitter.__init__` has required arguments.